<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExpertSystems2</class>
 <widget class="QMainWindow" name="ExpertSystems2">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>877</width>
    <height>567</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ExpertSystems2</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <property name="sizeConstraint">
         <enum>QLayout::SetMinimumSize</enum>
        </property>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QLabel" name="label_rules">
            <property name="text">
             <string>Используемые правила:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTableWidget" name="tableWidget_input_rules">
            <property name="sizePolicy">
             <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QPushButton" name="pushButton_add_rule">
              <property name="text">
               <string>Добавить</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_delete_rule">
              <property name="text">
               <string>Удалить</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_save_rules">
              <property name="text">
               <string>Сохранить</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Исходное выражение:</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLineEdit" name="lineEdit_in_STR"/>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_execute_DFS">
              <property name="text">
               <string>Выполнить</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <property name="sizeConstraint">
       <enum>QLayout::SetMaximumSize</enum>
      </property>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Вывод результатов:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTableWidget" name="tableWidget_out">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>877</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>tableWidget_input_rules</tabstop>
  <tabstop>pushButton_add_rule</tabstop>
  <tabstop>pushButton_delete_rule</tabstop>
  <tabstop>pushButton_save_rules</tabstop>
  <tabstop>lineEdit_in_STR</tabstop>
  <tabstop>pushButton_execute_DFS</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
